src/main.cpp

```diff
@@ -1,8 +1,23 @@
 #include <iostream>
+#include "../include/InventoryManager.hpp"
```

Comment:
Can this be ```#include "InventoryManager.hpp"``` rather than hard-coding the include path?  The makefile has the ```-I./include``` flag that specifies where include files are.